example/90-shoot.yaml

```diff
@@ -148,6 +148,17 @@ spec:
   # workersSettings:
   #   sshAccess:
   #     enabled: false
+# etcd:
```

Comment:
Can we add a short documentation for the new feature in `docs/development/autoscaling-specifics-for-components.md` (I am open for other location, I just couldn't find better one at first look)? It will be useful for linking the docs for operators and end users who will want to consume it.